cpp/src/toppra/geometric_path/piecewise_poly_path.hpp

```diff
@@ -29,11 +29,14 @@ class PiecewisePolyPath : public GeometricPath {
    *
    * See class docstring for details.
    *
-   * @param coefficients Polynoamial coefficients.
+   * @param coefficients Polynomial coefficients.
    * @param breakpoints Vector of breakpoints.
    */
   PiecewisePolyPath(const Matrices &coefficients, std::vector<value_type> breakpoints);
 
+
+  PiecewisePolyPath(const Vectors &positions, const Vector &times, const std::array<BoundaryCond, 2> &bc_type);
```

Comment:
Please add docstring (compatible with doxygen) to describe this constructor.

Reply:
Noted. Added in https://github.com/hungpham2511/toppra/pull/161/commits/9023de3c13e92777776f6dd9b0b05fb78f2fbe84. Thanks.